packages/store-engine-sqleet/src/index.ts

```diff
@@ -89,62 +78,29 @@ export class SQLeetEngine implements CRUDEngine {
     await this.isSupported();
 
     this.storeName = storeName;
-
-    let existingDatabase: Uint8Array | undefined = undefined;
-    if (this.rawDatabase) {
-      existingDatabase = await this.load(this.rawDatabase);
-      this.rawDatabase = undefined;
-    }
-
-    const SQL = await initSqlJs(this.dbConfig);
-    this.db = new SQL.Database(existingDatabase);
-
-    // Settings
-    this.db.run('PRAGMA `encoding`="UTF-8";');
-    this.db.run(`PRAGMA \`key\`=${escape(this.encryptionKey)};`);
+    await this.db.mount({key: this.encryptionKey}, this.storeName);
 
     // Create tables
     let statement = '';
-
     for (const tableName in this.schema) {
       const table = this.schema[tableName];
       statement += createTableIfNotExists(tableName, table);
     }
-
-    this.db.run(statement);
+    await this.db.run(statement);
 
     return this.db;
   }
 
-  public async export<T>(): Promise<string> {
-    if (!this.db) {
-      throw new Error('SQLite needs to be available');
-    }
-    const database: Uint8Array = new Uint8Array(this.db.export());
-    const strings = [];
-    const chunkSize = 0xffff;
-    // There is a maximum stack size. We cannot call `String.fromCharCode` with as many arguments as we want
-    for (let i = 0; i * chunkSize < database.length; i++) {
-      strings[i] = String.fromCharCode.apply(null, <any>database.subarray(i * chunkSize, (i + 1) * chunkSize));
-    }
-    return strings.join('');
-  }
-
-  private async load(database: string): Promise<Uint8Array | undefined> {
-    const databaseBinary = new Uint8Array(database.length);
-    for (let i = 0; i < database.length; i++) {
-      databaseBinary[i] = database.charCodeAt(i);
-    }
-    return databaseBinary;
+  async export<T>(): Promise<string> {
+    return (this.db.export('utf8') as unknown) as string;
```

Comment:
```suggestion
    return this.db.export('utf8') as any;
```

Reply:
I think here it's ok to cast into `string` since the function returns `Promise<Uint8Array | string>` and `any` would be to broad in that case.